packages/query/tsconfig.json

```diff
@@ -6,7 +6,8 @@
     "tsBuildInfoFile": "dist/.tsbuildinfo",
     "rootDir": "src",
     "outDir": "./dist",
-    "lib": ["dom"]
+    "lib": ["dom"],
```

Comment:
Can you please try removing this line and see if it produces any errors. The target for this isn't dom